articles/logs/streams/datadog.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Datadog Event Log Streams
+description: Datadog Event Log Streams let you export your events in near real-time to Datadog.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Datadog Event Log Streams
+
+Datadog Event Log Streams let you export your log events to Datadog for further analysis and monitoring. When Auth0 creates a log entry for your tenant, a copy is automatically sent to Datadog with the `source` and `service` both set to `auth0`.
+
+## Create a Datadog Event Stream
+
+### Copy API Key from Datadog
+
+1. Head to the Datadog dashboard.
+2. Navigate to **Integrations** -> **APIs**.
+![Integrations Dashboard](/media/articles/logs/datadog/tutorial-1.png)
+3. Expand the API Keys section on this page, then copy the API Key that you would like to use.
+![API Keys Section](/media/articles/logs/datadog/tutorial-2.png)
+
+### Configure settings in Auth0
+
+1. Log in to the [Auth0 Dashboard](${manage_url}).
+2. Navigate to **Logs > Streams**.
+3. Click **+ Create Stream**.
+4. Select **Datadog** and enter a unique name for your new Datadog Event Stream.
+5. On the next screen, provide the following settings for your Datadog Event Stream:
```

Comment:
I think step 5 should be more consistent to how we explain it for Eventbridge

Reply:
@camicano , I need to provide a little more explanation about each field here which is why I need the table. Is that ok?

Reply:
Also, this is consistent with the HTTP docs:

https://docs-content-staging-pr-8792.herokuapp.com/docs/logs/streams/http-event